pages/community/events/index.js

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, { useState } from 'react';
+import Button from '../../../components/buttons/Button';
+import GoogleCalendarButton from '../../../components/buttons/GoogleCalendarButton';
+import ICSFileButton from '../../../components/buttons/ICSFileButton';
+import Heading from '../../../components/typography/Heading';
+import { ArrowRightIcon } from '@heroicons/react/outline';
+import EventPostItem from '../../../components/navigation/EventPostItem';
+import EventFilter from '../../../components/navigation/EventFilter';
+import GenericLayout from '../../../components/layout/GenericLayout';
+import Meeting from '../../../components/Meeting';
+import NewsletterSubscribe from '../../../components/NewsletterSubscribe';
+import TextLink from '../../../components/typography/TextLink';
+import meetings from '../../../config/meetings.json';
+
+
+import Paragraph from '../../../components/typography/Paragraph';
+
+function index() {
+  const image = '/img/social/website-card.png';
+  const [events, setEvents] = useState(
+    meetings
+      ? meetings.sort((i1, i2) => {
+          const i1Date = new Date(i1.date);
+          const i2Date = new Date(i2.date);
+
+          if (i1.featured && !i2.featured) return -1;
+          if (!i1.featured && i2.featured) return 1;
+          return i2Date - i1Date;
+        })
+      : meetings
+  );
+
+  return (
+    <GenericLayout
+      title="AsyncAPI events"
+      description="Our catalogs of events and meetups"
+      image={image}
+      hideBanner={true}
+      wide
+    >
+      <div className="mt-10 sm:mt-28">
+        <div className="w-full sm:w-9/12">
+          <h1 className="countdown-text-gradient font-semibold text-3xl lg:text-7xl leading-tight md:text-4xl">
+            Join an AsyncAPI event from anywhere in the world.
+          </h1>
+          <div className="mt-10">
+            <GoogleCalendarButton href="https://calendar.google.com/calendar/u/3?cid=Y19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t" />
+            <ICSFileButton
+              href="https://calendar.google.com/calendar/ical/c_q9tseiglomdsj6njuhvbpts11c%40group.calendar.google.com/public/basic.ics"
+              className="mt-2 md:mt-0 md:ml-2"
+            />
+          </div>
+        </div>
+        <div className="flex justify-end mt-10">
+          <div className="w-[80%] sm:w-1/3">
+            <Heading
+              level="h2"
+              typeStyle="body-lg"
+              textColor="text-gray-700"
+              className="text-slate-500"
+            >
+              “All events/meetings are live streamed to all AsyncAPI social
```

Comment:
it looks weird on the website, that `"` are there, suggests that it is a quote from somewhere